packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -2289,6 +2298,35 @@ export class StripeHelper {
       this.expandResource(invoice.charge, CHARGES_RESOURCE),
     ]);
 
+    // if the invoice does not have the deprecated discount property but has a discount ID in discounts
```

Comment:
Is the `discount` property meant to be deprecated?

Reply:
It is marked as deprecated in the stripe API docs but `discounts` is not expanded by default.